src/App.tsx

```diff
@@ -30,6 +32,12 @@ const connector = new AppSearchAPIConnector({
     }),
 });
 
+/* const connector = new AppSearchAPIConnector({
+  engineName: "my-advisory-mentorship-data",
+  endpointBase: "http://localhost:3002",
+  searchKey: "search-2kdkz1y911uherajaewizm4v",
+}); */
+
 const configurationOptions = {
```

Comment:
Please remove all comments in this object, think they're all quite self-explanatory if one were to check the documentation.